docs/source/getting-started/tabular-data/grouping.ipynb

```diff
@@ -875,7 +873,7 @@
     "The mean is computed by dividing separate sum and length estimates.\n",
     "Since the partition length is marked as public information, the length is released in the clear, without any noise.\n",
     "\n",
-    "Reading the table, the smallest suitable noise scale parameter for the sum is 5762. \n",
+    "Reading the table, the smallest suitable noise scale parameter for the sum is 7056. \n",
```

Comment:
Preference to keep exact numbers out of the human-written text: It's really easy to re-evaluate the notebook and forget to make the update here.

Reply:
(But this would be a slightly bigger re-write, so it doesn't need to be in this PR.)